requirements.adoc

```diff
@@ -6,7 +6,11 @@ Before you install the multicluster engine for Kubernetes operator 1.0, review t
 * <<supported-operating-systems-and-platforms,Supported operating systems and platforms>>
 * <<network-configuration,Network configuration>>
 
-*Note:* There is no console for the Technical Preview version of this product.
+*Important:* Install multicluster engine for Kubernetes on a cluster that is not a Red Hat Advanced Cluster Management for Kubernetes cluster. The multicluster engine for Kubernetes cannot co-exist with Red Hat Advanced Cluster Management for Kubernetes because multicluster engine for Kubernetes and Red Hat Advanced Cluster Management provide some of the same management components.
+
+*Best practice:* It is recommended that you install multicluster engine for Kubernetes on a cluster that has never previously installed Red Hat Advanced Cluster Management.
+
+*Note:* There is no console for the Technical Preview version of this technology.
```

Comment:
I don't know why this Note is necessary.  Having a console or not is not a requirement for an operator, so I don't know why we would need to point out MCE doesn't have a console.  Especially in a way, as worded, that suggests this might be "fixed" in post-Tech Preview.  I suggest we delete this sentence.